docs/core/porting/upgrade-assistant-wpf-framework.md

```diff
@@ -11,83 +11,434 @@ The [.NET Upgrade Assistant](upgrade-assistant-overview.md) is a command-line to
+
+```
+[16:10:42 INF] Applying upgrade step Back up project
+Please choose a backup path
+   1. Use default path [C:\code\migration\wpf\sampleApp.backup]
+   2. Enter custom path
+```
 
-:::image type="content" source="media/upgrade-assistant-wpf-framework/update-tfm.png" alt-text=".NET Upgrade Assistant update TFM":::
+The tool chooses a default backup path named after the current folder, but with `.backup` appended to it. You can choose a custom path as an alternative to the default path. For each upgraded project, the folder of the project is copied to the backup folder. In this example, the `WebSiteRatings` folder is copied from _sampleApp\WebSiteRatings_ to _sampleApp.backup\WebSiteRatings_ during the backup step:
+
+```
+[16:10:44 INF] Backing up C:\code\migration\wpf\sampleApp\WebSiteRatings to C:\code\migration\wpf\sampleApp.backup\WebSiteRatings
+[16:10:45 INF] Project backed up to C:\code\migration\wpf\sampleApp.backup\WebSiteRatings
+[16:10:45 INF] Upgrade step Back up project applied successfully
+Please press enter to continue...
+```
+
+#### Upgrade the project file
+
+The project is upgraded from the .NET Framework project format to the .NET SDK project format.
+
+```
+[16:10:51 INF] Applying upgrade step Convert project file to SDK style
+[16:10:51 INF] Converting project file format with try-convert, version 0.3.261602+8aa571efd8bac422c95c35df9c7b9567ad534ad0
+[16:10:51 INF] Recommending TFM net6.0-windows because of dependency on project C:\code\migration\wpf\sampleApp\StarVoteControl\StarVoteControl.csproj
+C:\code\migration\wpf\sampleApp\WebSiteRatings\WebSiteRatings.csproj contains a reference to System.Web, which is not supported on .NET Core. You may have significant work ahead of you to fully port this project.
+C:\code\migration\wpf\sampleApp\WebSiteRatings\WebSiteRatings.csproj contains an App.config file. App.config is replaced by appsettings.json in .NET Core. You will need to delete App.config and migrate to appsettings.json if it's applicable to your project.
+[16:10:52 INF] Converting project C:\code\migration\wpf\sampleApp\WebSiteRatings\WebSiteRatings.csproj to SDK style
+[16:10:53 INF] Project file converted successfully! The project may require additional changes to build successfully against the new .NET target.
+[16:10:55 INF] Upgrade step Convert project file to SDK style applied successfully
+Please press enter to continue...
+```
+
+Pay attention to the output of each step. Notice that example output indicates that you'll have a manual step to complete after the upgrade:
+
+> App.config is replaced by appsettings.json in .NET Core. You will need to delete App.config and migrate to appsettings.json if it's applicable to your project.
+
+#### Move NuGet references
+
+Once the project format has been updated, the next step is to move the NuGet package references from the _packages.config_ file to the project file.
```

Comment:
Technically, the references are moved as part of the previous step. This step is about cleaning up the references because _packages.config_ listed all the NuGet dependencies needed whereas `<PackageReference>` style references only need to list top-level dependencies. This step will remove non-top-level (transitive) dependencies, and make other 'clean up' swaps like using the System.Configuration.ConfigurationManager package instead of the System.Configuration framework assembly.